deep_speech_2/model.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, vocab_size, num_conv_layers, num_rnn_layers,
                              rnn_layer_size)
         self._create_parameters(pretrained_model_path)
         self._inferer = None
+        self._loss_inferer = None
```

Comment:
No big difference between loss and cost. I prefer loss for it is more commonly used.